hack/test/pre-upgrade-setup.sh

```diff
@@ -29,7 +29,7 @@ spec:
     type: Image
     image:
       ref: ${TEST_CATALOG_IMG}
-      pollInterval: 24h
+      pollIntervalMinutes: 24
```

Comment:
That will break the e2e tests. 
Indeed it is 24 hours and not minutes.

Reply:
Updated.

Reply:
Note: this test is broken since #1439. We need another release to make it work again.